recommenders/popularity_recommender.py

```diff
@@ -1,31 +1,27 @@
-from recommenders import data_movies, data_ratings, default_poster_url
-import pandas as pd
-
-
-def recommend_popular_movies():
-
-    ######################################################################################################
-    movie_ratings = None
-    '''
```

Comment:
The variable 'movie_ratings' is initialized to None, but it is not used before reassigning its value in line 11. Consider removing the unnecessary initialization in line 9.